src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -486,6 +496,10 @@ public function merge(ContainerBuilder $container)
         $this->addAliases($container->getAliases());
         $this->getParameterBag()->add($container->getParameterBag()->all());
 
+        if (!$this->context->isLocked()) {
+            $this->context->merge($container->getContext());
```

Comment:
It happens, in the [MergeExtensionConfigurationPass](https://github.com/ogizanagi/symfony/blob/99e2273de29c558659aa53f3a6f520eae19c1c1a/src/Symfony/Component/DependencyInjection/Compiler/MergeExtensionConfigurationPass.php#L57), where the context is locked and the container not frozen.

See https://travis-ci.org/ogizanagi/symfony/jobs/152795773#L2232